### ISSUE TITLE
Hiding action drop-down menu(Explorer)

### ISSUE THREAD

**Issue by FuzzyAzurik** [CAAL/CAAL#48]
There should be a dropdown menu, where the user can deselect/select some action. By deselecting an action it will hide the action from the arbor-graph(it will rename to tau action?)

**Comment by senevoldsen90** [comment]
I do not think this is possible in this manner:  what will happen with tau actions leading to deadlock or livelock? What about the user adds a relabelling to tau for uninteresting actions and then collapse?
